index.js

```diff
@@ -64,25 +47,7 @@ module.exports = function ({ content, filename, sourceMap }) {
       + transpile(`${exportsTarget}.staticRenderFns = [${staticRenderFns}];`)
       + `\n${exportsTarget}.render._withStripped = true;`;
   }
-  const result = `${scriptPart}${compiledTemplate}`;
```

Comment:
Just debugged a few components using this method and it works probably even better than with source maps.